src/ducks/transactions/Transactions.styl

```diff
@@ -26,16 +26,16 @@
     .bnk-transaction-mobile
         & > td
             width 100%
+            padding-top .75rem
+            padding-bottom .75rem
```

Comment:
why is this necessary ?

Reply:
Because the base vertical padding on tables is `1rem`, but here we want the padding to be a little bit smaller.